templates/admin/base.html

```diff
@@ -0,0 +1,112 @@
+{% load staticfiles %}{% load i18n admin_static %}<!DOCTYPE html>
```

Comment:
Die Änderungen würde ich in einem separaten Branch machen. Das wird im Moment noch nicht benötigt und es müssten noch einige Anpassungen gemacht werden, sodass es gut benutzbar wäre.